nodejs-week1/avg.js

```diff
@@ -0,0 +1,19 @@
+const args = process.argv.slice(2);
+const numbers = args.map(Number);
+
+
+function calculateAverage(numbers) {
+    const sum = numbers.reduce((a, b) => a + b, 0);
+    return sum / numbers.length;
+}
+
+if (numbers.includes(0)) {
+    console.log("Add list of NUmber");
+} else if (numbers.some(isNaN)) {
+    console.log("Error: All arguments must be valid numbers.")
+}else {
+    const average = calculateAverage(numbers);
+    console.log(`The average is: ${average}`);
+}
+
+
```

Comment:
Good job!

Reply:
thank you for the review, I will do that